pkgs/development/gnuradio-modules/grnet/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, mkDerivation
+, fetchFromGitHub
+, gnuradio
+, cmake
+, pkg-config
+, boost
+, log4cpp
+, python
+, swig
+, mpir
+, gmp
+, doxygen
+, libpcap
+, icu
+, thrift
+}:
+
+let
+  version = {
+    "3.7" = "beb1cd75d006a982c0a9536e923800c5a0575451";
+    "3.8" = "b02016043b67a15f27134a4f0b0d43f5d1b9ed6d";
+    "3.9" = "e6dfd140cfda715de9bcef4c1116fcacfeb0c606";
+  }.${gnuradio.versionAttr.major};
+  src = fetchFromGitHub {
+    owner = "ghostop14";
+    repo = "gr-grnet";
+    rev = "${version}";
+    sha256 = {
+      "3.7" = "0000000000000000000000000000000000000000000=";
```

Comment:
You forgot this.

Reply:
Seems like the dependency `python27Packages.jinja2` is fixed. I'll rebase to the master branch and fill this.